sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample7_ConsumeEvents.cs

```diff
@@ -0,0 +1,132 @@
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Samples.Infrastructure;
+
+namespace Azure.Messaging.EventHubs.Samples
+{
+    /// <summary>
+    ///   An introduction to consuming events, using a simple <see cref="EventHubConsumer" />.
+    /// </summary>
+    ///
+    public class Sample7_ConsumeEvents : ISample
+    {
+        /// <summary>
+        ///   The name of the sample.
+        /// </summary>
+        ///
+        public string Name { get; } = nameof(Sample7_ConsumeEvents);
+
+        /// <summary>
+        ///   A short description of the sample.
+        /// </summary>
+        ///
+        public string Description { get; } = "An introduction to consuming events, using a simple Event Hub consumer.";
+
+        /// <summary>
+        ///   Runs the sample using the specified Event Hubs connection information.
+        /// </summary>
+        ///
+        /// <param name="connectionString">The connection string for the Event Hubs namespace that the sample should target.</param>
+        /// <param name="eventHubName">The name of the Event Hub, sometimes known as its path, that she sample should run against.</param>
+        ///
+        public async Task RunAsync(string connectionString,
+                                   string eventHubName)
+        {
+            // An Event Hub consumer is associated with a specific Event Hub partition and a consumer group.  The consumer group is
+            // a label that identifies one or more consumers as a set.  Often, consumer groups are named after the responsibility
+            // of the consumer in an application, such as "Telemetry" or "OrderProcessing".  When an Event Hub is created, a default
+            // consumer group is created with it, called "$Default."
+            //
+            // Each consumer has a unique view of the events in the partition, meaning that events are available to all consumers
+            // and are not removed from the partition when a consumer reads them.  This allows for different consumers to read and
+            // process events from the partition at different speeds and beginning with different events without interfering with
+            // one another.
+            //
+            // When events are published, they will continue to exist in the partition and be available for consuming until they
+            // reach an age where they are older than the retention period.
+            // (see: https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-faq#what-is-the-maximum-retention-period-for-events)
+            //
+            // Because events are nor removed from the partition when consuming, a consumer must specify where in the partition it
+            // would like to begin reading events.  For example, this may be starting from the very beginning of the stream, at an
+            // offset from the beginning, the next event available after a specific point in time, at a specific event.
+
```

Comment:
> Because events are nor removed from the partition

typo: nor -> not

> specific point in time, at a specific event.

add in 'or' -> specific point in time, or at a specific event.